### PR TITLE
Users table

### PR DESCRIPTION
**Pull request by emanuelmetal**
Added Users table as Home Page with redux and bootstrap styles.

### CODE REVIEW

File: webapp/app/actions/authActions.test.js [hunk 1]
```diff
@@ -1,16 +1,17 @@
 import configureMockStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 import * as authActions from './authActions';
-import * as types from '../constants/actionTypes';
+import actionTypes from '../constants/actionTypes';
 
+const { AUTH } = actionTypes;
```
Comment (javier-pepe) [inline webapp/app/actions/authActions.test.js:6]: QUESTION: do we need this intermediate variable here? 

PREF: destructure it in the import instead ( if you are looking for clarity, maybe assign it a new name :authActionTypes ?)

`import { AUTH: authActionTypes } from '../constants/actionTypes';
...
type: authActionTypes.LOGIN_REQUEST
`

File: webapp/app/components/pages/UsersForm/UsersForm.jsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+    actions: bindActionCreators(authActions, dispatch)
+  };
+}
+
+export default connect(mapStateToProps, mapDispatchToProps)(HomePage);
```
Comment (mravinale) [inline webapp/app/components/pages/UsersForm/UsersForm.jsx]: what do you think if we add test to this new component?

Comment (emanuelmetal) [reply webapp/app/components/pages/UsersForm/UsersForm.jsx]: On schedule!! Task is on Trello.

File: webapp/app/actions/authActions.test.js (hunk 1 above)
Comment (emanuelmetal) [reply webapp/app/actions/authActions.test.js:6]: `Import` can't destructure objects, are Just Named Imports, you need to import the object and then destructure it.
By the way Syntax is wrong it shall be
```
import { NamedImport as Alias } from 'foo'
```

File: webapp/app/components/pages/UsersForm/UsersForm.jsx (hunk 2 above)
Comment (emanuelmetal) [reply webapp/app/components/pages/UsersForm/UsersForm.jsx]: Done

**Comment by emanuelmetal** [comment]
@mravinale, @rootsco This is ready, I will add more UT on [This story](https://trello.com/c/R27dhDfL/10-add-missing-ut-for-the-current-component-configuration)

File: webapp/app/actions/authActions.test.js (hunk 1 above)
Comment (javier-pepe) [reply webapp/app/actions/authActions.test.js:6]: I started adding that reply to your changes to do it like this

const { AUTH: authAuctionTypes } syntax is correct there, then i switched to the import and left that one there, thanks

File: webapp/app/components/pages/UsersForm/UsersForm.jsx (hunk 2 above)
Comment (mravinale) [reply webapp/app/components/pages/UsersForm/UsersForm.jsx]: thanks, let's remember to include UT or integration test inlcuded in the same feature, helps code review and agile development process.

**Review by javier-pepe — APPROVED** [review]

**Review by mravinale — APPROVED** [review]

**Pull request merged**